src/components/Session/index.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import Button from "react-bootstrap/Button";
+import Card from "react-bootstrap/Card";
+import { selectUser } from "../../store/user/selectors";
+import { utcToZonedTime } from "date-fns-tz";
+import { useSelector, useDispatch } from "react-redux";
+import { bookSession, unBookSession } from "../../store/session/actions";
+import { NavLink } from "react-router-dom";
+
+
+export default function Session({ session, teacher, btn, hideSubject,bg }) {
+  const startDate = utcToZonedTime(session.start_date, "Europe/Berlin")
+    .toString()
+    .slice(0, 21);
+  const endDate = utcToZonedTime(session.end_date, "Europe/Berlin")
```

Comment:
I'm guessing this timezone later will be dynamic? Timezones are a MESS, I encourage you to take a look at momentjs